backend/server/register.py

```diff
@@ -0,0 +1,17 @@
+import sqlalchemy
+from flask import Blueprint, request, redirect, jsonify
+
+from domain.use_cases.admin_usecases import AdminRegister
+
+register_blueprint = Blueprint('register_blueprint', __name__)
+
+
+@register_blueprint.route('/register', methods=['POST'])
+def registration():
+    email = request.form['email']
+    psw = request.form['psw']
+    try:
+        AdminRegister(email, psw).admin_register()
+        return redirect('/login/', code=401)
```

Comment:
So here we either return 401 (Unautorised) or return and error json. 
That looks strange. Shouldn't one path be successful?
Shouldn't redirection return 2xx code?

Reply:
Also in swagger API we have 200 for successful registration.